tiled/_tests/test_hdf5.py

```diff
@@ -136,18 +136,20 @@ def test_from_file_with_empty_data(example_file_with_empty_data, buffer, key):
 
 
 @pytest.mark.filterwarnings("ignore: The dataset")
-@pytest.mark.parametrize("key", ["int", "float", "str", "bytes", "bool"])
-def test_from_file_with_scalars(example_file_with_scalars, buffer, key):
-    """Serve a single HDF5 file at top level."""
+@pytest.mark.parametrize("key", ["bool", "int", "float", "str", "bytes"])
+@pytest.mark.parametrize("num", [1, 5])
+def test_from_file_with_scalars(example_file_with_scalars, buffer, key, num):
```

Comment:
```suggestion
@pytest.mark.parametrize("key", ["bool", "int", "float", "str", "bytes"])
@pytest.mark.parametrize("num", [1, 5])
def test_from_file_with_scalars(example_file_with_scalars, buffer, key: str, num: int):
```

While Dan is away you're at the mercy of me insisting on type hints ;)